docs/API.md

```diff
@@ -89,7 +89,7 @@ XMLReport
 
 ## Configuration
 
-Given you have the following configuration file called `config.reek` in your root directory:
+Given you have the configuration file called `.reek.yml` in your root directory:
```

Comment:
I think it's clearer with the word 'following' included, or some other indication that what follows is the contents of `.reek.yml`.

Reply:
ok